### PR TITLE
Install phpmd using pear

### PR DESCRIPTION
**Pull request by jayvdb**
Closes https://github.com/coala/docker-coala-base/issues/167

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -123,11 +126,8 @@ RUN zypper addrepo http://download.opensuse.org/repositories/home:illuusio/openS
     perl-X11-Protocol \
     postfix \
     php7-zlib \
-    python-cssselect \
```
Comment (Vamshi99) [inline Dockerfile:126]: I think this is unrelated change 😕

Comment (jayvdb) [reply Dockerfile:126]: We have two build problems. Need both fixed for a greeh build

**Comment by Makman2** [comment]
ack e82da14 40fdd03

**Comment by Makman2** [comment]
@rultor merge
